### ISSUE TITLE
Relative path to global data dir with custom input dir

### ISSUE THREAD

**Issue by rolandtoth** [11ty/eleventy#655]
I'm trying to set the data directory to be in the root, so I set it to ```../data```, but apparently it doesn't support relative paths, at least not outside of the input dir. All data in ```data/cfg.json``` comes out as undefined:

```javascript
dir: {
      input: "input",
      data: "../data",
      output: "dist",
      includes: "../includes"
    }
```

Is this supposed to be this way or am I missing something? I would like to have only my posts in the input dir, and data dir in the root. I tried with 0.8.2 and 0.9.0 Beta 1 as well.

**Comment by octoxalis** [comment]
If `dir.input` is "`input`", your current directory when invoking eleventy command ( e.g. `npx eleventy`) have to be `input`.
Have a look at [INPUT DIRECTORY](https://www.11ty.io/docs/config/#input-directory) in Docs.

**Comment by rolandtoth** [comment]
Thanks, but I still wasn't able to move the data directory outside of the input dir. Anyway, no biggie.

**Comment by MadeByMike** [comment]
Is this now closed by #1280?

**Comment by zachleat** [comment]
Just tested this and the following config works in 1.0:

```
module.exports = function(eleventyConfig) {
  return {
    dir: {
      input: "input",
      data: "../data",
    }
  };
};
```

![image](https://user-images.githubusercontent.com/39355/154374223-9fa13b0c-4c08-4bc6-aeab-ecbb9fc25dbc.png)
